src/engine/tools/engine-suite/src/engine_test/cmds/run.py

```diff
@@ -10,6 +10,7 @@
 DEFAULT_POLICY = "policy/wazuh/0"
```

Comment:
Eventually there is an error when creating the session in the API-TEST:

```
root@wazuh-engine:~/repos/wazuh/src/engine# echo "test" | engine-test run syslog
The session could not be created. Error: unpack requires a buffer of 4 bytes
```
This happens with different event formats.